STM32Resources/BuildEnvironment_Setup/BuildEnvironment_Setup.adoc

```diff
@@ -0,0 +1,81 @@
+
+
+= *Setting up and Using an STM32 Build Environment on Linux*
+A guide for setting up a linux system to build projects for STM32
+
+
+This guide assumes you are starting with a Windows 64 bit system and intend to use a virtual machine in Oracle Virtualbox. I installed Linux Mint 18.3 If you already have a linux system set up, are using a Mac, or want to set it up a diffeent way figure out what parts of this guide you have already completed yourself and rejoin at a later step.
+
+== Install Virtualbox
+Download https://www.virtualbox.org/wiki/Downloads[Oracle Virtual Box]
+The latest version at writting was 5.2.4. Also download the VirtualBox Extension Pack from the same page.
+Install both.
+
+==== Setup a Virtual Machine
+Create a new virtual machine in Virtualbox.
+
+image::https://github.com/StudentOrganisationForAerospaceResearch/DevelopmentResources/blob/garnettanner/stm32-example/STM32Resources/BuildEnvironment_Setup/images/1.PNG[]
+
+There isn't an option or Linux Mint OS so choose Ubuntu.
```

Comment:
I would provide the information that Linux Mint is based off of Ubuntu, so the selected OS can be Ubuntu.